src/olympia/constants/permissions.py

```diff
@@ -61,6 +61,9 @@
 # Can moderate add-on ratings submitted by users.
 RATINGS_MODERATE = AclPermission('Ratings', 'Moderate')
 
+# Can access advanced reviewer features.
+REVIEWS_EDIT = AclPermission('Reviews', 'Edit')
```

Comment:
This description is a little vague, and the naming implies they can actually edit reviewer reviews (which I don't think it what it allows)

Reply:
Yeeeah I also find the name confusing. It's from one of the issues but I guess we can change it if we have a better one to suggest?

I really wasn't inspired for the comment, but I figured it was better than nothing :) Also open to ideas there.

Reply:
`Reviews: Admin`?  Still not great, admittedly.

Reply:
2025e66198edb860e69cde4c5b0028649d237d9e